packages/mattermost-redux/src/actions/users.ts

```diff
@@ -170,7 +178,7 @@ function completeLogin(data: UserProfile): ActionFunc {
 
         const serverVersion = Client4.getServerVersion();
         dispatch(setServerVersion(serverVersion));
-        if (!isMinimumServerVersion(serverVersion, 4, 7) && getConfig(getState()).EnableCustomEmoji === 'true') {
+        if (!isMinimumServerVersion(serverVersion, 4, 7) && getConfig(state).EnableCustomEmoji === 'true') {
```

Comment:
Not familiar with this part, but if `dispatch(getClientConfig())` on [L176](https://github.com/mattermost/mattermost-webapp/pull/8413/files#diff-24ee706807fb27c790f1a6a954a5aa1a581d4ae80f971cd8742c54f9f1cfe4ccR176) gets a different `config` version we'll need the `getState()` or we will be stuck with the old one.

Reply:
Yep, good call